web-ui/src/context/AppContext.js

```diff
@@ -155,7 +151,7 @@ const initialState = {
   isAdmin: false,
   mySkills: mySkills,
   teamMembers: teamMembers,
-  user: defaultUser
+  user: defaultUser,
```

Comment:
Unnecessary comma

Reply:
This is Prettier formatting that will happen every time I save

Reply:
Yeah, I wouldn't worry about this. Unless we want to redo the prettier config, which I think isn't worth it.